### PR TITLE
📖 Updated wording on prerequisites doc

### PR DESCRIPTION
**Pull request by MarcusNoble**
**What this PR does / why we need it**:

Updated details on prerequisites page to be more in line with current approach:
	- Removed assumption of bootstrap user
	- Updated SSH instructions to be optional
Made some small styling tweaks

### CODE REVIEW

File: docs/book/src/topics/using-clusterawsadm-to-fulfill-prerequisites.md [hunk 1]
```diff
@@ -145,20 +152,21 @@ The current iteration of the Cluster API Provider AWS relies on credentials
...
-export AWS_SECRET_ACCESS_KEY=$(echo $AWS_CREDENTIALS | jq .AccessKey.SecretAccessKey -r)
+export AWS_REGION=us-east-1 # This is used to help encode your environment variables
+export AWS_ACCESS_KEY_ID=<your-access-key>
+export AWS_SECRET_ACCESS_KEY=<your-secret-access-key>
+export AWS_SESSION_TOKEN=<session-token> # If you are using Multi-Factor Auth.
 ```
 
-If you did not use `clusterawsadm` to provision your user, you will need to set
-these environment variables in your own way.
+**Note**: The credentials used must have the appropriate permissions for use by the controllers.
```
Comment (sedefsavas) [inline docs/book/src/topics/using-clusterawsadm-to-fulfill-prerequisites.md:164]: So that account also need to have permissions for control plane policy?

Comment (MarcusNoble) [reply docs/book/src/topics/using-clusterawsadm-to-fulfill-prerequisites.md:164]: The permissions are needed for `capa-controller-manager` (and `capa-eks-control-plane-controller-manager` if EKS enabled). 
The credentials are used to populate these two secrets:
* `capa-eks-control-plane-controlplane-manager-bootstrap-credentials`
* `capa-manager-bootstrap-credentials`

**Comment by MarcusNoble** [comment]
Updated examples to now include the `--config` flag added in https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/2147

**Comment by richardcase** [comment]
@MarcusNoble - if you can squash the commits i can approve (i think)

**Comment by MarcusNoble** [comment]
Done :)

**Pull request merged**